chapters/functions.tex

```diff
@@ -1712,14 +1712,14 @@ \section{External Function Interface}\label{external-function-interface}
 This is just as for any other function.  The components in the protected part allow local variables for temporary storage to be declared.
 \end{nonnormative}
 
-The \lstinline!language-specification! must currently be one of \lstinline!"builtin"!, \lstinline!"C"!, \lstinline!"C..."! (for one of the specific C standards like C89, C99, and C11 -- specifying
+The \lstinline!language-specification! must currently be one of \lstinline!"builtin"! (deprecated), \lstinline!"C"!, \lstinline!"C..."! (for one of the specific C standards like C89, C99, and C11 -- specifying
```

Comment:
I only understood that we deprecated that these functions were builtin - not the entire concept of external "builtin".

Reply:
OK.  If that was unclear I suggest that we ask for a clarifying poll between meetings.

To me, it seems strange to keep the concept if we don't see a use for it (which would be why it was deprecated for the current set of functions).  Another reason to get rid of the unused concept is that the concept uses problematic terminology; it doesn't match my expectations at all that we would have a language without any built-in functions.

Reply:
But if you instead skip that deprecated and replace the next one by something like the following:

`The \lstinline!"builtin"! specification is deprecated for the only case where it is specified: the elementary mathematical functions described in \cref{built-in-mathematical-functions-and-external-built-in-functions}.`

To me that deprecation is as strong.

Reply:
It is not as strong, because only deprecating the current uses does neither allow simplification of terminology (I'd like to consider all of the functions described in the specification as "built in"), nor point in the direction of a Modelica language simplification.

However, given that there seems to have been misunderstandings regarding what we were polling on in the meeting, I still think we should do another poll – it wouldn't be right to describe `"builtin"` as deprecated even if I was able to convince you, as long as this is not how the poll was understood by the rest of the group.